client/src/components/files/FolderSelectionModal.vue

```diff
@@ -64,21 +75,25 @@
         v-if="currentEntries.length > 0"
       >
         <ion-item
-          class="file-list-item"
+          class="file-item"
           v-for="entry in currentEntries"
           :key="entry[0].id"
           :disabled="entry[1]"
           @click="enterFolder(entry[0])"
         >
-          <div class="file-name">
-            <div class="file-name__icons">
-              <ms-image
-                :image="entry[0].isFile() ? getFileIcon(entry[0].name) : Folder"
-                class="file-icon"
-              />
-            </div>
-            <ion-label class="file-name__label cell">
-              {{ entry[0].name }}
+          <div class="file-item-image">
+            <ms-image
+              :image="entry[0].isFile() ? getFileIcon(entry[0].name) : Folder"
+              class="file-item-image__icon"
+            />
+          </div>
+          <ion-label class="file-item__name cell">
+            {{ entry[0].name }}
+          </ion-label>
+          <!-- last update -->
+          <div class="file-lastUpdate">
```

Comment:
Same here, this information is not visible in small display, yet it is not stated explicitly in the code.